### PR TITLE
perf: reduce supported uptimes

### PR DESCRIPTION
**Pull request by czarcas7ic**
Closes: #7537

## What is the purpose of the change

When calling updateAccumulatorsToNow, we update all 6 accumulators, despite not using the 1W and 2W accumulators. We should remove these for now, and if we decide we need these later we can implement them at that time (and maybe just enable at the per pool level, so we don't need to go through this unnecessary overhead).

## Summary by CodeRabbit

- **Refactor**
	- Simplified and optimized uptime calculation logic for better performance.
- **New Features**
	- Introduced a new query for all before send hooks, enhancing the app's functionality.
- **Bug Fixes**
	- Removed duplicate logic and entries to ensure data consistency.
- **Tests**
	- Adjusted test scenarios to reflect changes in uptime calculations and incentive distribution, ensuring accurate testing environments.
- **Documentation**
	- Updated test case descriptions to reflect changes in time durations and incentive calculations.

### CODE REVIEW

**Review by ValarDragon — COMMENTED** [review]
Reviewed everything except incentives test! Lgtm thus far.

We're sure e2e is making cl positions lre upgrade and swapping in that pool pre + post upgrade right? And claiming?

Just want to make sure there's no edge case around this possible

**Review by PaddyMc — APPROVED** [review]

**Review by mattverse — APPROVED** [review]
LGTM

File: x/concentrated-liquidity/incentives_test.go [hunk 1]
```diff
@@ -2844,7 +2840,7 @@
 			growthOutside:    uptimeHelper.twoHundredTokensMultiDenom,
 			numShares:        osmomath.OneDec(),
 
-			expectedError: types.NegativeDurationError{Duration: time.Hour * 336 * -1},
+			expectedError: types.NegativeDurationError{Duration: time.Hour * 648 * -1},
```
[the remark this answers is not in the document]
Comment (czarcas7ic) [reply x/concentrated-liquidity/incentives_test.go:2843]: Negative duration is tested with defaultTime being set to false

File: x/concentrated-liquidity/incentives_test.go [hunk 2]
```diff
@@ -3373,8 +3369,8 @@
 			expectedAuthorized:     time.Nanosecond,
 		},
 		"Unordered authorized uptimes": {
-			preSetAuthorizedParams: []time.Duration{time.Hour * 24 * 7, time.Nanosecond, time.Hour * 24},
-			expectedAuthorized:     time.Hour * 24 * 7,
+			preSetAuthorizedParams: []time.Duration{time.Hour, time.Nanosecond, time.Hour * 24},
+			expectedAuthorized:     time.Hour * 24,
```
[the remark this answers is not in the document]
Comment (czarcas7ic) [reply x/concentrated-liquidity/incentives_test.go:3373]: Its okay for now

File: x/concentrated-liquidity/incentives_test.go [hunk 3]
```diff
@@ -1758,8 +1758,8 @@ func (s *KeeperTestSuite) TestInitOrUpdatePositionUptimeAccumulators() {
 func (s *KeeperTestSuite) TestQueryAndCollectIncentives() {
 	ownerWithValidPosition := s.TestAccs[0]
 	uptimeHelper := getExpectedUptimes()
+	oneHour := time.Hour
```
[the remark this answers is not in the document]
Comment (czarcas7ic) [reply x/concentrated-liquidity/incentives_test.go:1761]: We are asserting bank balances, which asserts the incentives are claimed

**Pull request merged**